rpcs3/Crypto/unself.cpp

```diff
@@ -1343,6 +1343,27 @@ static bool IsSelfElf32(const fs::file& f)
 	return (elf_class[4] == 1);
 }
 
+static bool IsDebugSelf(const fs::file& f)
```

Comment:
you can just add a bool template param to CheckDebugSelf and early out with return true if the param is met

Reply:
> 

A what :D ?